Docs/source/running_cpp/parameters.rst

```diff
@@ -281,6 +281,14 @@ Particle initialization
       and optional argument ``<species_name>.do_symmetrize`` (whether to
       symmetrize the beam in the x and y directions).
 
+    * ``external_file``: Inject macroparticles with properties (charge, mass, position, and momentum) according to data read from external openPMD file.
+      It requires the additional arguments:
+      ``<species_name>.injection_file`` (`string`) name of the openPMD file and
+      ``<species_name>.physical_q_tot`` (`double`) optional (default is no weighting, ``weight=q_p``) when specified, WarpX injects macroparticles with the same properties as the sample of ``N`` physical particles, each with charge ``q_p``, described in the file, except their weight. Their weight is computed via ``weight=<species_name>.physical_q_tot/q_p/N``, so that the injected macroparticles represent the whole physical species.
+      When using this style, it is not necessary to add other ``<species_name>.(...)`` paramters, because they will be read directly from the file.
```

Comment:
if you specify the default above more precisely, this sentence is probably not needed anymore.
Typo:  parameters